src/main/java/picard/nio/PicardBucketUtils.java

```diff
@@ -92,22 +89,24 @@ public static java.nio.file.Path getPathOnGcs(String gcsUrl) {
     }
 
     /**
+     * Note: to the extent possible, avoid converting an HtsPath object to a String then calling this method.
+     * Instead, use the same method below that takes in a PicardHtsPath as input.
+     *
      * @param path path to inspect
      * @return true if this path represents a gcs location
      */
-    public static boolean isGcsUrl(final String path) {
+    private static boolean isGcsUrl(final String path) {
         GATKUtils.nonNull(path);
-        return path.startsWith(GCS_PREFIX);
+        return path.startsWith("gs://");
```

Comment:
use the `GOOGLE_CLOUD_STORAGE_FILESYSTEM_SCHEME` constant here (similar to the isHadoopUrl(String) one is written).

Reply:
done